src/xst.md

```diff
@@ -1,3 +1,90 @@
+
+### Stablecoins and Synthetic Assets Today
+
+Currently, there are 5 different types of stablecoins/pegged assets:
+
+- **Fiat-Backed Stablecoins (or Tokenized Fiat):** These stablecoins are backed by and pegged to dollars (or other fiat currency), their value remains tied to the price of the pegged currency. For example, [USDC](https://www.circle.com/en/usdc).
+- **Crypto-Backed Stablecoins (or On-Chain Collateralized Stablecoins):** These are backed by other crypto assets. For example [DAI](https://makerdao.com/en/).
+- **Precious Metal-Backed Stablecoins (or Off-Chain Collateralized
+  Stablecoins):** Like gold-standard fiat, these stablecoins use gold and other metals to back their value. For example [Tether Gold](https://gold.tether.to/).
+- **Algorithmic Stablecoins:** These stablecoins use algorithms to back their value, there are some variants that can be pegged to fiat values, depending on the algorithm used. For example [AMPL](https://www.ampleforth.org/).
+- **Synthetic assets** are tokens with value pegged to an oraclized asset. Purely synthetic assets can be used to represent many types of value, such as, to track the shares of financial instruments and securities. For example, [SYNTHETIX](https://synthetix.io/).
+
+XST is an algorithmic stablecoin for the SORA ecosystem, initially proposed in 2018, based on the ideas of Irving Fisher's compensated dollar. However, instead of being backed by gold, XST assets are backed by XOR.
+
+# What Makes SORA XST Different?
+
+Now that stablecoins and synthetic assets have been defined and classified, you might be wondering what makes XST stand out from the rest?
+
+XST helps to solve the problem of XOR liquidity by creating synthetic assets backed by a variable amount of XOR and pegged to a target index (e.g., a currency). XOR is minted/deminted to always guarantee the value of the pegged index. The first index implemented is linked to the value of DAI and will be called XSTUSD.
+
+### Non-Collateralized Stability
+
+In the case of XST assets, as they are backed by XOR, XOR can be algorithmically minted or deminted (burned) to provide the full value for the XST assets upon demand. The XST Primary Market Maker that mints/demints XST assets and XOR is built in as a liquidity source into Polkaswap's liquidity aggregator, so buyers/sellers will always get the full value in XOR for their XST assets, and the price will never deviate (lower or higher) from the asset price. Therefore, XST is not subject to price slippage or a lack of liquidity. This works, for example, in the case of XSTUSD, because a single XSTUSD is a claim for $1 USD _worth of XOR_, and **not a claim for actual $USD itself**.
```

Comment:
```suggestion
In the case of XST assets, as they are backed by XOR, XOR can be algorithmically minted or deminted (burned) to provide the full value for the XST assets upon demand. The XST Primary Market Maker that mints/demints XST assets and XOR is built as a liquidity source into Polkaswap's liquidity aggregator. This means that buyers and sellers will always get the full value in XOR for their XST assets, and the price will never deviate (lower or higher) from the asset price. Therefore, XST is not subject to price slippage or a lack of liquidity. This works, for example, in the case of XSTUSD, because a single XSTUSD is a claim for $1 USD _worth of XOR_, and **not a claim for actual $USD itself**.
```